help/xdm/home.md

```diff
@@ -115,7 +115,7 @@ Real-Time Customer Profile consumes schema-formatted data based on the [!UICONTR
 
 >[!NOTE]
 >
->Real-Time Customer Profile does **not** support schemas based on classes other than the [!UICONTROL XDM ExperienceEvent] class.
+>Real-Time Customer Profile does **not** support schemas based on classes other than the [!UICONTROL XDM Individual Profile] and [!UICONTROL XDM ExperienceEvent] classes.
```

Comment:
Hi Diagant. Can you confirm whether this statement is accurate please?

Reply:
Digant said that AFAIK, we do support other classes via lookups. This statement seems to be bit incorrect and it may be better to remove it entirely. I will make this change.